plugins/catalog/src/overridableComponents.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Overrides } from '@material-ui/core/styles/overrides';
+import { StyleRules } from '@material-ui/core/styles/withStyles';
+
+import { EntityLinksEmptyStateClassKey } from './components/EntityLinksCard';
+import { SystemDiagramCardClassKey } from './components/SystemDiagramCard';
+
+type PluginCatalogComponentsNameToClassKey = {
+  PluginCatalogEntityLinksEmptyState: EntityLinksEmptyStateClassKey;
+  PluginCatalogSystemDiagramCard: SystemDiagramCardClassKey;
+};
+
+export type BackstageCoreComponentsOverrides = Overrides & {
```

Comment:
So 2 options here, should we call this `BackageCatalogComponentsOverrides` or should we always call this type `BackstageOverrides` for each package?

Reply:
Oh, missed this somehow. I actually like your proposal of just going with `BackstageOverrides` and that will avoid having to update the `core-components` package with this noop.

Reply:
Yep, I think it's fine to just have to rename them when importing them `as Catalog` or whatever,